core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -2250,14 +2250,40 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
           .setScope(Scope.MASTER)
           .build();
+  /**
+   * @deprecated use {@link #MASTER_INODE_METASTORE} and {@link #MASTER_BLOCK_METASTORE} instead
+   */
+  @Deprecated
```

Comment:
I actually have a random idea. Can we add an alias to the property: alluxio.master.inode.metastore and introduce a new one for block: alluxio.master.block.metastore. So that the user can
1. just specify alluxio.master.metastore for both
2. specify alluxio.master.inode.metastore AND alluxio.master.block.metastore

@jiacheliu3 wdty. @maobaolong please ignore this comment for now. Not sure this is a better way or not. :)

Reply:
+1